docs/sources/panels-visualizations/visualizations/datagrid/index.md

```diff
@@ -0,0 +1,82 @@
+---
+aliases:
+  - ../../features/panels/datagrid_panel/
+  - ../../reference/datagrid/
+  - ../../visualizations/datagrid/
+description: Learn about datagrid panel visualization features.
+keywords:
+  - grafana
+  - dashboard
+  - panels
+  - datagrid
+  - datagrid panel
+  - datagrid options
+menuTitle: Datagrid
+title: Datagrid
+weight: 950
+---
+
+# Datagrid
+
+> **Note:** Grafana Datagrid panel is Experimental. This feature is supported by the engineering team on a best-effort basis, and breaking changes may occur without notice prior to general availability.
+
+The datagrid panel offers a user the ability to create or edit and fine-tune data within Grafana. As such, this panel can act as a data source for other panels
+inside a dashboard. Let us dive deeper into the flows and functionalities of this panel.
+
+![Datagrid panel](/media/docs/datagrid/screenshot-grafana-datagrid-panel.png)
+
+## Use cases
+
+The datagrid panel can be used to manipulate data queried from any data source or from a dragged and dropped file. It can then be used as a simple tabular
```

Comment:
```suggestion
The datagrid panel can be used to manipulate data queried from any data source, or from a dragged and dropped file. It can then be used as a simple tabular
```

Reply:
I'm curious about the rules here. I would normally write without a comma, is it because the second part after the `or`is an independent clause? Would writing it this way, without the `or` be correct? - `queried from any data source or dragged and dropped file`

Reply:
The comma here indicates a pause, which I think clarifies the distinctions between the 2 options ("any data source" || "drag and dropped file"). But, again, no strong opinion either way; I think both with and without the comma are correct.